### PR TITLE
app setup proposition

### PR DESCRIPTION
**Pull request by hl**
first app / api setup

### CODE REVIEW

File: app/controllers/site_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # SiteController handles the frontend for the website
 class SiteController < ApplicationController
   def index
+    # show the front page
+  end
+
+  def redirect
```
Comment (jeroenvisser101) [inline app/controllers/site_controller.rb:7]: I'd suggest the sitecontroller is used for the 'sessionless' site only, with only static content.

**Comment by jeroenvisser101** [comment]
On the topic of retrieving products, I would like to be able to retrieve the products using the image's hashes. I'd sent them, either in a GET or POST as an array of hashes as such: `["a3ebff...", "a3ebff...", ...]`

**Comment by hl** [comment]
Check `api_controller.rb`

**Comment by jeroenvisser101** [comment]
Ah great :)

Will we use the hashes as primary keys?

**Comment by jeroenvisser101** [comment]
toch nie? 😂

**Pull request closed without merging**